catalog.html

```diff
@@ -1,306 +1,323 @@
+        </p>
+        <a href="">Оформить заказ</a>
       </div>
-    </div>
-    <nav>
-      <ul>
-        <li>
-          <a href="index.html">Главная</a>
-        </li>
-        <li>
-          <a href="">Компания</a>
-        </li>
-        <li>
-          <a>Каталог</a>
-        </li>
-        <li>
-          <a href="">Новости</a>
-        </li>
-        <li>
-          <a href="">Спецпредложения</a>
-        </li>
-        <li>
-          <a href="">Доставка</a>
-        </li>
-        <li>
-          <a href="">Контакты</a>
-        </li>
-      </ul>
-    </nav>
-  </header>
 
-  <main>
-    <nav>
-      <ul>
-        <li>
-          <a href="index.html">Главная</a>
-        </li>
-        <li>
-          <a>Каталог</a>
-        </li>
-        <li>
-          <a href="">Инструмент</a>
-        </li>
-        <li>
-          Перфораторы
-        </li>
-      </ul>
-    </nav>
-
-    <h1>Перфораторы</h1>
-    <form action="https://echo.htmlacademy.ru/courses" method="get">
-      <h2>Фильтр:</h2>
-      <fieldset>
-        <legend>Цена:</legend>
-        <input type="range" min="0" max="30000" step="500">
-      </fieldset>
-      <fieldset>
-        <legend>Производители:</legend>
+      <div>
+        <p>
+          Интернет-магазин строительных материалов и инструментов для ремонта
+        </p>
+        <p>
+          <a href="tel:+78125550555">+7(812)555-05-55</a>
+        </p>
+        <p>
+          г.Санкт-Петербург, ул.Б.Конюшенная, д.19/8
+        </p>
+        <div>
+          <a href="">Равшан Джамшутович</a>
+          <a href="" aria-label="Выход"></a>
+          <a href="">Мои заказы</a>
+          <a href="">Личный кабинет</a>
+        </div>
+      </div>
+      <nav>
         <ul>
           <li>
-            <label>
-              <input type="checkbox" name="bosch">
-              Bosch
-            </label>
+            <a href="index.html">Главная</a>
+          </li>
+          <li>
+            <a href="">Компания</a>
           </li>
           <li>
-            <label>
-              <input type="checkbox" name="interskol">
-              Интерскол
-            </label>
+            <a>Каталог</a>
           </li>
           <li>
-            <label>
-              <input type="checkbox" name="makita">
-              Makita
-            </label>
+            <a href="">Новости</a>
           </li>
           <li>
-            <label>
-              <input type="checkbox" name="dewalt">
-              Dewalt
-            </label>
+            <a href="">Спецпредложения</a>
           </li>
           <li>
-            <label>
-              <input type="checkbox" name="hitachi">
-              Hitachi
-            </label>
+            <a href="">Доставка</a>
+          </li>
+          <li>
+            <a href="">Контакты</a>
           </li>
         </ul>
-      </fieldset>
-      <fieldset>
-        <legend>Питание:</legend>
+      </nav>
+    </header>
+
+    <main>
+      <nav aria-label="breadcrumbs">
```

Comment:
```suggestion
      <nav aria-label="навигация хлебными крошками">
```

тоже, нужен перевод, тк у твоей страницы русский язык (html lang="ru")

вот ариа свойства, которые переводятся: https://w3c.github.io/aria/#translatable-states-and-properties